packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx

```diff
@@ -92,11 +92,27 @@ export const RemoteScreenShare = React.memo(
       return (
         <VideoTile
           renderElement={
+            // Add the layer on top of the pptlive stream to diable keyboard event
+            // TODO need to remove after we have navigation feature ready
```

Comment:
// TODO: Remove after we have navigation feature ready
// [ADO work item link here]